managed/src/main/java/com/yugabyte/yw/cloud/CloudAPI.java

```diff
@@ -44,4 +44,12 @@ public CloudAPI get(String code) {
     Provider provider,
     Map<Region, Set<String>> azByRegionMap,
     Set<String> instanceTypesFilter);
+
+  /**
+   * Check whether AWS credentials(AWS access key and Secret Key) are valid or not.
+   *
+   * @param config The AWS credentials info.
+   * @return true if credentials are valid otherwise return false.
+   */
+  boolean isAWSValidCredentials(Map<String, String> config);
```

Comment:
As CloudAPI.java is a common class (it is situated outside of the `aws` folder) then it is assumed to have no specific methods (related to exact cloud types). Let's imagine that this interface is implemented for Azure, as example, which also has a check for valid credentials.
My proposal - to rename the function, to remove AWS mentions from the comment.

Reply:
Make sense! I had seen CloudAPI.java was not being used anywhere other than AWSCloudImpl.java.

Will rename the method as you suggested! Thanks!

Reply:
Done